flow/project.py

```diff
@@ -1490,7 +1491,17 @@ def run(self, jobs=None, names=None, pretend=False, np=None, timeout=None, num=N
             Show a progress bar during execution.
         :type progess:
             bool
+        :param order:
+            Specify the order of operations, possible values are:
+                * None or 'none' (no reordering, do not assume any order)
+                * 'by-job'  (operations are grouped by job)
+                * 'round-robin' (order operations in round-robin fashion by job)
```

Comment:
Alternatively this could be called 'cyclic' -- to me, 'round-robin' _could_ suggest a finite time slot offered to each operation, as described here: https://en.wikipedia.org/wiki/Round-robin_scheduling

It's clear enough as-is, just wanted to offer this as a suggestion in case you like it better.

Reply:
I've switched it to cyclic.